### PR TITLE
TerminalFont: support Windows Terminal on WSL

### PR DESCRIPTION
**Pull request by CarterLi**
I still hope fastfetch can support Windows some day.

My few efforts: https://github.com/CarterLi/fastfetch/commit/2a65be0afd12837ebe92c4812c0e9ef6fe1637ff

### CODE REVIEW

File: src/data/config_user.txt [hunk 1]
```diff
@@ -324,3 +324,4 @@
 #--lib-osmesa /usr/lib/libOSMesa.so
 #--lib-opencl /usr/lib/libOpenCL.so
 #--lib-plist /usr/local/bin/libplist-2.0.dylib
+#--lib-plist /usr/lib/libcjson.so
```
Comment (LinusDierheimer) [inline src/data/config_user.txt:327]: #--lib-cjson

**Review by LinusDierheimer — COMMENTED** [review]
Please update the GH actions file too, to build with libcjson support in the linux build.

**Comment by LinusDierheimer** [comment]
Thanks again!

**Comment by CarterLi** [comment]
Well adding dependency `libcjson-dev` didn't seem to make cmake find libcjson on Ubuntu, maybe

```diff
diff --git a/CMakeLists.txt b/CMakeLists.txt
index f157d0b..3256cb1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -376,7 +376,7 @@ ff_lib_enable(GLX glx)
 ff_lib_enable(OSMESA osmesa)
 ff_lib_enable(OPENCL OpenCL)
 ff_lib_enable(LIBPLIST libplist-2.0)
-ff_lib_enable(LIBCJSON libcjson)
+ff_lib_enable(LIBCJSON libcjson libcjson1)
```

**Comment by LinusDierheimer** [comment]
https://packages.ubuntu.com/focal/amd64/libcjson-dev/filelist

Ubuntu starts shipping the package config file with 20.08, and gh uses 20.04, so no chance for us to build with support sadly.

**Pull request merged**